v0/destinations/engage/data/EngageIdentifyConfig.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "sourceKeys": ["traits.email", "context.traits.email"],
+    "destKey": "email"
+  },
+  {
+    "sourceKeys": "firstName",
+    "destKey": "first_name",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "lastName",
+    "destKey": "last_name",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "phone",
+    "destKey": "number",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "originalTimestamp",
+    "destKey": "created_at"
+  },
+  {
+    "sourceKeys": ["externalId.engageListId", "config.listIds"],
+    "destKey": "lists"
+  }
```

Comment:
As far as I remember, config JSONs are only built to fetch data from event.message, not event.destination. Have we tested it? Also, `externalId.engageListId` will not recieve array of values in `value` key. So that implementation needs to be thought through.

Reply:
Oh yes I was just testing it forgot to remove it.